src/index.js

```diff
@@ -37,7 +37,7 @@ const createGroups = function (Bottleneck, common) {
   })
 }
 
-function throttlingPlugin (octokit, octokitOptions = {}) {
+export default function throttlingPlugin (octokit, octokitOptions = {}) {
```

Comment:
I would prefer to `export function throttlingPlugin`, it's easier for compatibility between ES Modules and commonJS. You’ll have to update the README imports as well

```js
import { pluginThrottling } from "https://unpkg.com/@octokit/plugin-throttling";
```

Sorry that I only found out about this now